Library/Homebrew/utils/inreplace.rb

```diff
@@ -39,21 +39,22 @@ def initialize(errors)
     # @api public
     sig {
       params(
-        paths:        T.any(T::Array[T.untyped], String, Pathname),
+        paths:        T.any(T::Enumerable[T.any(String, Pathname)], String, Pathname),
         before:       T.nilable(T.any(Pathname, Regexp, String)),
         after:        T.nilable(T.any(Pathname, String, Symbol)),
         audit_result: T::Boolean,
       ).void
     }
     def inreplace(paths, before = nil, after = nil, audit_result = true) # rubocop:disable Style/OptionalBooleanParameter
+      paths = Array(paths)
       after &&= after.to_s
       before = before.to_s if before.is_a?(Pathname)
 
       errors = {}
 
-      errors["`paths` (first) parameter"] = ["`paths` was empty"] if paths.blank?
```

Comment:
Some slight logic change to account for the fact that `Enumerable` doesn't necessarily define `blank?`. This is technically more strict (`['']` would previously have been ok), but i believe to be semantically correct.